tests/zfs-tests/tests/functional/cli_root/zfs_property/zfs_written_property_001_pos.ksh

```diff
@@ -120,7 +120,7 @@ sync_pool
 written=$(get_prop written $TESTPOOL/$TESTFS1)
 writtenat3=$(get_prop written@snap3 $TESTPOOL/$TESTFS1)
 [[ $written -eq $writtenat3 ]] || \
-    log_fail "Written and written@ dont match $written $writtenat3"
+    log_fail "Written and written@ don't match $written $writtenat3"
```

Comment:
Do we normally use contractions in these? Should we? How about "do not"

Reply:
> Do we normally use contractions in these? Should we? How about "do not"

You tell me. If you want I can change it.

Reply:
I grepped. There are lots of contractions.